src/components/Filters.vue

```diff
@@ -1,64 +0,0 @@
-<template>
-  <div>
-    <div class="filter-species">
-      <h4 class="text-left">Species</h4>
-      <div class="form-check text-left" v-for="specie in species" :key="specie">
-        <input
-          class="form-check-input"
-          type="checkbox"
-          :value="specie"
-          name="filterSpecies"
-          v-model="filterSpecies"
-          @change="onSpecieChange()"
```

Comment:
use `v-on`. That way you don't need to explicitly define native events.

Reply:
I have to pass 2 onChange events to this component from the parent, hence used custom events and emitted the values.